flink-rpc/flink-rpc-akka-loader/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaRpcSystemLoader.java

```diff
@@ -42,6 +43,15 @@
  */
 public class AkkaRpcSystemLoader implements RpcSystemLoader {
 
+    /**
+     * The name of the akka dependency jar, which should be correspond to the artifact output copied
+     * to flink-runtime module.
```

Comment:
```suggestion
     * The name of the akka dependency jar, which must match the jar that is bundled by the flink-rpc-akka-loader module.
```
We no longer copy a jar to flink-runtime.